css/css-display/accessibility/display-contents-role-and-label.html

```diff
@@ -116,7 +116,7 @@ <h2>Accessibility roles</h2>
     <h2>x</h2>
     <ul style="display: contents" data-expectedrole="list" data-testname="ul with display: contents, as child of div with display: grid, has list role" class="ex-role">
       <li>x</li>
-      <li data-expectedrole="listitem" data-expectedlabel="x" data-testname="listitem within ul with display: contents, as child of div with display: grid, has listitem role" class="ex-role-and-label">x</li>
+      <li data-expectedrole="listitem" data-testname="listitem within ul with display: contents, as child of div with display: grid, has listitem role" class="ex-role">x</li>
```

Comment:
Reviewers, hopefully the diff is self-explanatory, but to explain the line-by-line CI results, you'll see the following:

- 9 new role tests in the first section (testnames without a Label: or Role: prefix), then 
- 9 missing results in the "Label:"-prefixed test list, then 
- 9 missing results in the "Role:"-prefixed test list. 

HTH with the one confusing part of an otherwise straightforward review.